include/boost/geometry/algorithms/detail/buffer/turn_in_piece_visitor.hpp

```diff
@@ -19,33 +19,34 @@
 
 #include <boost/geometry/algorithms/comparable_distance.hpp>
 #include <boost/geometry/algorithms/covered_by.hpp>
-#include <boost/geometry/algorithms/detail/disjoint/point_box.hpp>
+#include <boost/geometry/algorithms/detail/buffer/buffer_policies.hpp>
 #include <boost/geometry/algorithms/detail/disjoint/box_box.hpp>
+#include <boost/geometry/algorithms/detail/disjoint/point_box.hpp>
 #include <boost/geometry/algorithms/detail/dummy_geometries.hpp>
-#include <boost/geometry/algorithms/detail/buffer/buffer_policies.hpp>
 #include <boost/geometry/geometries/box.hpp>
 
-
-namespace boost { namespace geometry
+namespace boost
+{
+namespace geometry
 {
 
 #ifndef DOXYGEN_NO_DETAIL
 
-namespace detail { namespace buffer
+namespace detail
+{
+namespace buffer
 {
 
-template
-<
-    typename CsTag,
-    typename Turns,
-    typename Pieces,
-    typename DistanceStrategy,
-    typename UmbrellaStrategy
+template <typename CsTag,
+          typename Turns,
+          typename Pieces,
+          typename DistanceStrategy,
+          typename UmbrellaStrategy
```

Comment:
This is also different than what we had. But it is (as far as I can see) the variant closest possible.